src/ui.rs

```diff
@@ -1132,7 +1396,25 @@ impl App {
             match &self.input_mode {
                 InputMode::Editing => {
                     f.set_cursor(
-                        chunks[2].x + self.input.len() as u16 + 10, // 10 is for "Command: "
+                        chunks[2].x + self.input.len() as u16 + "Command (Tab for auto-complete, ↑/↓ for history, Ctrl+L to clear): ".len() as u16,
+                        chunks[2].y + 1,
+                    );
+                }
+                InputMode::QuickReply { target_peer } => {
+                    let prefix_len = format!("📤 Quick reply to {}: ", target_peer).len();
```

Comment:
More information pasted to the UI screen not using the Icons type for windows compatability